### PR TITLE
MF-162 - ES With NATS JetStream

### LINKED ISSUE #162: ES With NATS Jet Stream
**FEATURE REQUEST**

Document event streaming with NATS Jet Stream

### PR DESCRIPTION
**Pull request by Musilah**
### What does this do?
Add Nats Jetstream Event Sourcing Documentation to the Docs. 
### Which issue(s) does this PR fix/relate to?
 Resolves #162

### List any changes that modify/break current functionality
None
### Have you included tests for your changes?
No
### Did you document any new/modified functionality?
Yes.

### CODE REVIEW

**Review by drasko — DISMISSED** [review]
This is OK for me. @rodneyosodo @dborovcanin and @SammyOina please review.

File: docs/events.md [hunk 1]
```diff
@@ -638,191 +940,307 @@ By fetching and processing these events you can reconstruct `things` service sta
...
+        18) "lingering-sea"
+        19) "tags"
+        20) "[morning-pine]"
+        21) "created_at"
+        22) "2023-08-29T12:17:50.566453Z"
+    ```
+
+2. In Nats JetStreams
+
+    ```nats
```
Comment (dborovcanin) [inline docs/events.md]: Use `bash` or `json`.

File: docs/events.md [hunk 2]
```diff
@@ -638,191 +940,307 @@ By fetching and processing these events you can reconstruct `things` service sta
...
+    Subject: events.mainflux.things Received: 2023-10-05T18:38:40+03:00
+
+    {"created_at":"2023-10-05T15:38:40.264564Z","id":"47540f84-029b-436f-89b5-3c10f87e302b","metadata":"eyJVcGRhdGUiOiJCZXJuYXJkLUJyaWNrZXkifQ==","name":"Bence-Jefferson","occurred_at":1696520320651750298,"operation":"thing.update_tags","owner":"3264e965-3fe5-4d4e-a857-48de43551d2e","status":"enabled","tags":"[Kac-Kimma]","updated_at":"2023-10-05T15:38:40.643285Z","updated_by":"3264e965-3fe5-4d4e-a857-48de43551d2e"}
+    ```
+
+3. In rabbitmq streams
+
+    ```
```
Comment (dborovcanin) [inline docs/events.md]: Use ```json.

File: docs/events.md (hunk 1 above)
Comment (dborovcanin) [reply docs/events.md]: Do that for all the other cases.

**Review by dborovcanin — CHANGES_REQUESTED** [review]

File: docs/events.md [hunk 3]
```diff
@@ -2044,5 +2044,5 @@ Example of disconnect event:
 
 
 [redis-streams]: https://redis.io/topics/streams-intro
-[mf-docker-compose]: https://github.com/mainflux/mainflux/blob/main/docker/docker-compose.yml
-[bootstrap-docker-compose]: https://github.com/mainflux/mainflux/blob/main/docker/addons/bootstrap/docker-compose.yml
+[mf-docker-compose]: https://github.com/mainflux/mainflux/blob/master/docker/docker-compose.yml
```
Comment (rodneyosodo) [inline docs/events.md]: Change the URLs

**Review by rodneyosodo — CHANGES_REQUESTED** [review]

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**